style.css

```diff
@@ -0,0 +1,74 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+.wrapper {
```

Comment:
- [ ] Your other linter error is that there should exist one empty line between this two, please add it to solve your issue.